WeTransferPRLinter/Tests/WeTransferPRLinterTests/XCResultSummaryReporterTests.swift

```diff
@@ -50,8 +50,10 @@ final class XCResultSummartReporterTests: XCTestCase {
         XCTAssertEqual(danger.fails.count, 2)
         let failure = try XCTUnwrap(danger.fails.first)
         XCTAssertEqual(failure.message, "**TestTests.testFailureJosh1():**<br/>XCTAssertTrue failed")
-        XCTAssertEqual(failure.file, "test-ios/TestTests/TestTests.swift")
-        XCTAssertEqual(failure.line, 36)
+
+        // Enable once inline reporting works again.
```

Comment:
It just occurred to me: should it be updated to a warning?

Reply:
Agreed! I rewrote this in an upcoming PR using expected failure:
<img width="992" alt="image" src="https://user-images.githubusercontent.com/4329185/163532867-74a0c1c1-4259-448b-9fe1-83992b4c400e.png">